components/scream/data/scream_input.yaml

```diff
@@ -39,10 +39,21 @@ Atmosphere Driver:
       - ch4
       - o2
       - n2
+      # Year to use for orbital calculations. If positive, use a fixed year for
```

Comment:
@jgfouca - I seem to remember something about our CIME setup not keeping comments from our YAML.  Is this an issue just with buildnl or would this chunk of comments in this PR also be impacted?

Reply:
@AaronDonahue , these comments will not appear in $case/run/scream_input.yaml. The new README file for documenting scream_input.yaml tells the user to look at this file in the repo if they want to see comments. I think that's the best we can do unless we want to return to the hacky system I had for trying to preserve these comments. It's possible I can come up with a smarter design than relying purely on line numbers which failed any time the processing of the file by buildnml changed the line count.

Reply:
Oh no worries.  I'm happy for the comments to get deleted in the actual case.  I just wanted to make sure that the existence of comments wouldn't cause an issue.  This is great.

Reply:
Imho, it's ok to keep comments only in the template input file.

Reply:
Absolutely.  Agreed.